### PR TITLE
Improving styling on new Jekyll IFRC Go website

### PR DESCRIPTION
**Pull request by KatHicks**
Added some detail to the README.md about installing all the packages and some good resources to use.

Updated banner picture, corrected isotope filtering, box styles and buttons.

Still need to figure out how the internal links work so as to add in the other pages (e.g. appeals, 3W, drefs etc).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +78,37 @@ The site will automatically refresh since it is bundled with livereload.
...
+> If you're developing a server application, please note that you need to listen to 0.0.0.0 ($IP) and 8080 ($PORT). 
+
+Follow these instructions to change the port and IP for gulp serve: <https://github.com/Swiip/generator-gulp-angular/issues/848>
+
+###### Step #1
+Changed the port in the gulpfile.js on line 107 as follows:
+```
+gulp.task('serve', ['build'], function () {
+  browserSync({
+    port: 8080,
```
Comment (dalekunce) [inline README.md]: Sorry going to be opinionated. This needs to stay on 3000.

File: README.md [hunk 2]
```diff
@@ -49,6 +78,37 @@ The site will automatically refresh since it is bundled with livereload.
 
 The `_config-dev.yml` file will be loaded alongside `_config.yml`.
 
+##### Running on Cloud9 IDE:
```
Comment (dalekunce) [inline README.md]: Please move all of the Cloud9 stuff to another readme.

File: README.md [hunk 3]
```diff
@@ -33,10 +37,35 @@ To set up the development environment for this website, you'll need to install t
...
+```
+
+##### Command #3: Installing Jeykll
+
+```
+$ gem install jekyll --no-rdoc --no-ri (Ubuntu)
 ```
-$ npm install
+
+##### Command #4: Installing earlier version of Chunky PNG (version 1.3.7)
```
Comment (dalekunce) [inline README.md]: remove from readme and address the root cause of the problem.

File: README.md [hunk 4]
```diff
@@ -33,10 +37,35 @@ To set up the development environment for this website, you'll need to install t
...
+```
+
+##### Command #2: Installing Bundler
+
+```
+$ bundle install (Mac)
+$ gem bundle install (Ubuntu)
+```
+
+##### Command #3: Installing Jeykll
```
Comment (dalekunce) [inline README.md]: this step is not needed if bundle install runs correctly. please remove.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 ## [IFRC GO LABS](http://ifrcgo.github.io)
 
+### Understanding Jekyll
```
Comment (dalekunce) [inline README.md]: remove please

**Comment by dalekunce** [comment]
@KatHicks the styling is for sure better. Want to meet later this week so we can do a code review on my stuff so you know how it works.

File: README.md (hunk 1 above)
Comment (KatHicks) [inline README.md]: yes, that's fine. just changed to 8080 as [cloud9](https://c9.io/) listens to port 8080. so for me to work on it, had to change.

Comment (KatHicks) [inline README.md]: @dalekunce - do you want me to change it back for each pull request?

File: README.md (hunk 2 above)
Comment (KatHicks) [inline README.md]: done!

File: README.md (hunk 5 above)
Comment (KatHicks) [inline README.md]: done!

**Pull request merged**